### DISCUSSION TITLE
如何把iptables 翻译成  nftables，以避免下载时的高负载。

### DISCUSSION THREAD

**Discussion by c71an** [vernesong/OpenClash#2701] [Q&A]
仅80和443端口走openclash：

iptables -t nat -D openclash -p tcp -j REDIRECT --to-ports 7892
iptables -t nat -A openclash -p tcp --dport 80 -j REDIRECT --to-ports 7892
iptables -t nat -A openclash -p tcp --dport 443 -j REDIRECT --to-ports 7892
 
https://github.com/vernesong/OpenClash/issues/101#issuecomment-614639470

**Comment by vernesong** [comment]
可以开仅常用端口
